dhis-2/dhis-services/dhis-service-node/pom.xml

```diff
@@ -48,7 +48,7 @@
     </dependency>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter</artifactId>
+      <artifactId>junit-jupiter-api</artifactId>
```

Comment:
JUnit suggests to use artifact `junit-jupiter`
https://github.com/junit-team/junit5-samples/blob/5f9e577e2c64b06f8b7ef5633462144feed97fef/junit5-jupiter-starter-maven/pom.xml#L29-L32

Is that still flagged by dependency analysis as unused?

Reply:
If I use `junit-jupiter` I have something like this in the dependency analysis:
`[WARNING] Used undeclared dependencies found:
[WARNING]    org.junit.jupiter:junit-jupiter-api:jar:5.8.2:test
[WARNING] Unused declared dependencies found:
[WARNING]    org.junit.jupiter:junit-jupiter:jar:5.8.2:test`